pages/community/precice-workshop.md

```diff
@@ -0,0 +1,101 @@
+
+<img class="img-responsive center-block" src="images/events/precice2023-group-inside.jpeg" alt="Group picture" style="width: 70%; margin:auto;">
+
+## More than a conference
+
+Of course, there are talks. We typically have a single track of user and developer talks, focusing on news, novel applications, and methods. At the same time, we give everyone the chance to introduce their research via quick introduction talks or other means (posters, forum posts).
+
+However, the real workshop experience begins after the talks.
+
+## Training
+
+Come to the preCICE workshop to learn how to run your first coupled simulations via an extensive [training course](community-training.html). This training course gets expanded every year, so it remains useful also for returning users as well.
+
+<img class="img-responsive center-block" src="images/events/precice2023-course.jpeg" alt="Training course" style="width: 70%; margin:auto;">
+
+## Shaping the future
+
+We call it a workshop to emphasize the interactivity. And one of the ways to interact is by participating to the World Café, in which users and developers discuss the next directions for the project, including how bigger features should work, and how community contributions could be better coordinated.
+
+<img class="img-responsive center-block" src="images/events/precice2023-world-cafe.jpeg" alt="World Café" style="width: 70%; margin:auto;">
+
+## Working together
+
+During the workshop, we reserve time to directly work with users on their applications.
+This offer is open to all workshop participants and the concrete planning and matching
+of users to experts happens during the workshop.
+We typically help new users design their coupling projects, discuss feature requests, or resolve bugs together.
+
+<img class="img-responsive center-block" src="images/events/precice2020-support.jpeg" alt="User support session" style="width: 70%; margin:auto;">
+
+## Giving back to the community
+
+The workshop is open to everyone, but we are aware that not everyone is able to attend,
+while we want to be able to refer to the content ourselves.
+For this reason, we release talk recordings on [YouTube](https://www.youtube.com/c/preCICECoupling), gradually after each workshop.
+
+We also publish [training material developed together with the community](https://github.com/precice/community-training).
+Note, however, that this cannot capture the hands-on collaborative experience that the workshop offers.
+
+## Small investment, large benefit
+
+We structure the workshop schedule in a way that you can attend selected parts of it,
+especially if you are already an experienced user, or if you just want to see
+what the latest updates and new applications are.
+
+We also set the participation fee just enough to cover the expenses,
+offering discounts for academia and supporting the participation
+of students and international researchers that would otherwise not be able to join.
```

Comment:
I am not sure here. This year we are using the fees to potentially support maintaining of preCICE. This would also be my long-term vision. Let's try to weaken the statement.